python/nav/web/macwatch/urls.py

```diff
@@ -17,14 +17,13 @@
 """macwatch Django URL config"""
 
 from django.urls import path
-from django.urls import re_path
 from nav.web.macwatch import views
 
 
 urlpatterns = [
     # Default view
     path('', views.list_watch, name='listwatch'),
     path('add/', views.add_macwatch),
-    re_path(r'^delete/(\d+)/$', views.delete_macwatch),
-    re_path(r'^edit/(\d+)/$', views.edit_macwatch),
+    path('delete/<int:id>/', views.delete_macwatch),
```

Comment:
The delete and edit path causes a TypeError, due to how Django deals with named parameters:

<img width="550" height="81" alt="Image" src="https://github.com/user-attachments/assets/39b4f9a9-b614-419f-8513-ef78aad48dfd" />

The view expects the parameter to have the same name, i.e. `macwatchid`. Changing the parameter name to `<int:macwatchid>` would fix this. 

<img width="380" height="57" alt="Image" src="https://github.com/user-attachments/assets/fbeda364-020c-4f20-adbc-2779fd8e203e" />